app/views/users/show.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>Users#show</h1>
```

Comment:
Please follow [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAAB34BLWS77NKSB5PMIHZJQ3GY4JGA2Q) wireframe for user's show page. The layout of the page should be the same as in the given wirefram.